### DISCUSSION TITLE
15w-suv-2024

### DISCUSSION THREAD

**Discussion by giscus** [bowencool/blog#80] [Announcements]
# 15w-suv-2024

15 万预算新能源 SUV 车型对比

https://blog.bowen.cool/posts/15w-suv-2024

**Comment by justde** [comment]
Hi 我想跟您咨询下，关于e5的续航达成率有多少呀，另外我看了测评悬挂软容易点头，使用过程中您觉得感知明显么？

**Reply by bowencool** [reply]
刚提车没几天，续航没仔细测。

点头确实有，感知明显，但不是特别严重。习惯后，只要轻踩刹车就可以避免。

**Reply by bowencool** [reply]
坐标昆明，全部城市道路，大多数时候表显 11～14kwh 每百公里

**Reply by bowencool** [reply]
高速满载（昆明到大理，前排2人，后排放平，塞满行李，还有车顶行李箱增加风阻和重量）定速 100km/h 电耗 16kwh/100km，相当可以了。当然这是气候优势，冷的地方还没去过。

**Comment by Aimerence** [comment]
也在看15万左右的车，对比 极狐S5/银河E8/领跑c11/哪吒L/小鹏MONA3，感觉极狐和领跑性价比可以，就怕后续保养维修更贵，当地店少。

**Reply by bowencool** [reply]
我看的都是SUV😁
